.github/workflows/build.yml

```diff
@@ -13,43 +13,26 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-      - uses: actions-rs/cargo@v1
-        with:
-          command: build
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - name: Checkout Repository
+        id: checkout
+        uses: actions/checkout@v5
 
-          # fmt:
-          #   name: Test
-          #   runs-on: ubuntu-latest
-          #   steps:
-          #     - uses: actions/checkout@v2
-          #     - uses: actions-rs/toolchain@v1
-          #       with:
-          #         profile: minimal
-          #         toolchain: stable
-          #         override: true
-          #     - uses: actions-rs/cargo@v1
-          #       with:
-          #         command: test
+      - name: Setup Rust
+        id: setup-rust
+        run: rustup update stable && rustup default stable && rustup component add rustfmt clippy
```

Comment:
Probably fine for now, but worth calling out that this likely will remain mismatched with the version used in the Dockerfile (at this time, latest stable version is `1.90`)
https://github.com/provenance-io/object-store/blob/36d6ec7c1711b3e55f7ee906e1f0dd835407a09f/Dockerfile#L1
So ideally either the workflow or the Dockerfile needs to change.